### PR TITLE
Adding the ability to start the petclinic client + a clustered active…

### PR DESCRIPTION
**Pull request by lanimall**
… / passive TSA

Hey so this pull request goes hand-in-hand with the other pull request for the pet clinic...
Basically, while setting this up, I realized that I could not spin up a full example with a clustered TSA directly from a docker-compose script...
So I created a new docker-compose-AandP.yml that allows that...and it uses a ENV variable to set the TSA url... (somehow, it seems like ENV variable do not get expanded in entrypoint["",""] so I added a new docker-entrypoint.sh as well to launch the pet clinic exec...
Long story short, I made it all work in my local...but wanted to pull it back to yours as I think it may be a useful example...

Let me know your thoughts!!

Fabien

### CODE REVIEW

**Comment by anthonydahanne** [comment]
Hello @lanimall !
Sorry for getting back to you so late !
Anyway, thanks for contributing your docker-compose.yml file, this is definitely something that's missing !
I'm currently in the process of moving everything to https://github.com/Terracotta-OSS/docker , so I won't merge your PR since I'm gonna re work a little bit the organization of the Dockerfiles, but I'll make sure to re use your docker-compose file !
Thanks again !
